b2sdk/progress.py

```diff
@@ -202,231 +197,3 @@ def make_progress_listener(description, quiet):
     else:
         return SimpleProgressListener(description)
 
-
```

Comment:
moving classes can be easily split to another PR (then review is way easier as reviewer knows you were not messing with the code)